src/core/SurfaceLayer.cpp

```diff
@@ -42,7 +42,7 @@ void ngen::SurfaceLayer::update_models(boost::span<double> catchment_outflows,
         double contribution_at_t = features.nexus_at(id)->get_downstream_flow(cat_id, current_time_index, 100.0);
 #if NGEN_WITH_ROUTING
         int nexus_index = nexus_indexes[id];
-        nexus_downstream_flows[nexus_index] = contribution_at_t;
+        nexus_downstream_flows[nexus_index] += contribution_at_t;
```

Comment:
Why was this and `catchment_outflows` changed from assignment to accumulation? I don't think there should be cases of multiple contributions